package/yast2-auth-server.spec

```diff
@@ -18,7 +18,7 @@
 Name:           yast2-auth-server
 Group:          System/YaST
 Summary:        A tool for creating identity management server instances
-Version:        4.2.4
+Version:        4.2.5
```

Comment:
This change is for SLE-15-SP3 so I'd expect the version should be 4.3.x

I really don't know why there is no 4.3.0 version :( @lslezak any idea?